Patch to change the she-bang from "/bin/csh" to "/usr/bin/env csh"
of all csh-scripts in the Ferret-7.6.0/bin/ directory, because tcsh/csh may
not be installed as RPM however is available in the $EBROOTGENTOO prefix.

Date:   2024-05-23
Author: Oliver Stueker <oliver.stueker@ace-net.ca>

diff -ru Ferret-7.6.0.orig/bin/Fapropos.csh Ferret-7.6.0/bin/Fapropos.csh
--- Ferret-7.6.0.orig/bin/Fapropos.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fapropos.csh	2024-05-23 18:16:39.114607229 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Fapropos
 #! scan the Ferret Users Guide for the indicated string
diff -ru Ferret-7.6.0.orig/bin/Fapropos.v491 Ferret-7.6.0/bin/Fapropos.v491
--- Ferret-7.6.0.orig/bin/Fapropos.v491	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fapropos.v491	2024-05-23 18:16:39.119607243 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Fapropos
 #! scan the Ferret Users Guide for the indicated string
diff -ru Ferret-7.6.0.orig/bin/Fapropos.v50 Ferret-7.6.0/bin/Fapropos.v50
--- Ferret-7.6.0.orig/bin/Fapropos.v50	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fapropos.v50	2024-05-23 18:16:39.123607255 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! Fapropos
 #! 6/99 *kob* Temp holder while V5.0 Documents are not available text format
 
diff -ru Ferret-7.6.0.orig/bin/Fdata.csh Ferret-7.6.0/bin/Fdata.csh
--- Ferret-7.6.0.orig/bin/Fdata.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fdata.csh	2024-05-23 18:16:39.133607284 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fdata file_template    
 # determine if FERRET data files matching file_template are currently on-line by
 # searching the paths in FER_DATA
diff -ru Ferret-7.6.0.orig/bin/Fdescr.csh Ferret-7.6.0/bin/Fdescr.csh
--- Ferret-7.6.0.orig/bin/Fdescr.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fdescr.csh	2024-05-23 18:16:39.142607310 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fdescr file_template    
 # determine if FERRET descriptor files matching template are currently on-line
 # by searching the paths in FER_DESCR
diff -ru Ferret-7.6.0.orig/bin/Fgo.csh Ferret-7.6.0/bin/Fgo.csh
--- Ferret-7.6.0.orig/bin/Fgo.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fgo.csh	2024-05-23 18:16:39.157607353 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fgo go_file_template    
 # determine if files matching go_file_template are currently on-line by
 # searching the paths in FER_GO
diff -ru Ferret-7.6.0.orig/bin/Fgrids.csh Ferret-7.6.0/bin/Fgrids.csh
--- Ferret-7.6.0.orig/bin/Fgrids.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fgrids.csh	2024-05-23 18:16:39.168607385 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fgrids grid_file_template    
 # determine if FERRET grid files matching template are currently on-line
 # by searching the paths in FER_GRIDS
diff -ru Ferret-7.6.0.orig/bin/Fhelp.csh Ferret-7.6.0/bin/Fhelp.csh
--- Ferret-7.6.0.orig/bin/Fhelp.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fhelp.csh	2024-05-23 18:16:39.177607410 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Fhelp
 #! enter the Ferret Users Guide at the indicated line number
diff -ru Ferret-7.6.0.orig/bin/Fhelp.v491 Ferret-7.6.0/bin/Fhelp.v491
--- Ferret-7.6.0.orig/bin/Fhelp.v491	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fhelp.v491	2024-05-23 18:16:39.180607419 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Fhelp
 #! enter the Ferret Users Guide at the indicated line number
diff -ru Ferret-7.6.0.orig/bin/Fhelp.v50 Ferret-7.6.0/bin/Fhelp.v50
--- Ferret-7.6.0.orig/bin/Fhelp.v50	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fhelp.v50	2024-05-23 18:16:39.185607433 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! Fhelp
 #! 6/99 *kob* Temp holder while V5.0 Documents are not available text format
 
diff -ru Ferret-7.6.0.orig/bin/Findex.csh Ferret-7.6.0/bin/Findex.csh
--- Ferret-7.6.0.orig/bin/Findex.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Findex.csh	2024-05-23 18:16:39.193607457 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Findex
 #! browse the Index of the Ferret Users Guide
diff -ru Ferret-7.6.0.orig/bin/Findex.v491 Ferret-7.6.0/bin/Findex.v491
--- Ferret-7.6.0.orig/bin/Findex.v491	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Findex.v491	2024-05-23 18:16:39.197607468 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Findex
 #! browse the Index of the Ferret Users Guide
diff -ru Ferret-7.6.0.orig/bin/Findex.v50 Ferret-7.6.0/bin/Findex.v50
--- Ferret-7.6.0.orig/bin/Findex.v50	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Findex.v50	2024-05-23 18:16:39.201607480 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! Findex
 #! 6/99 *kob* Temp holder while V5.0 Documents are not available text format
 
diff -ru Ferret-7.6.0.orig/bin/Finstall.csh Ferret-7.6.0/bin/Finstall.csh
--- Ferret-7.6.0.orig/bin/Finstall.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Finstall.csh	2024-05-23 18:16:39.210607506 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh
+#! /usr/bin/env csh
 # Finstall
 # Does two things:
 # 1) modifies 'ferret_paths template' to specify locations of ferret software &
diff -ru Ferret-7.6.0.orig/bin/Finstall.irix6 Ferret-7.6.0/bin/Finstall.irix6
--- Ferret-7.6.0.orig/bin/Finstall.irix6	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Finstall.irix6	2024-05-23 18:16:39.214607517 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Finstall
 # Does two things:
 # 1) modifies 'ferret_paths template' to specify locations of ferret software &
diff -ru Ferret-7.6.0.orig/bin/Finstall.pre_v62 Ferret-7.6.0/bin/Finstall.pre_v62
--- Ferret-7.6.0.orig/bin/Finstall.pre_v62	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Finstall.pre_v62	2024-05-23 18:16:39.223607543 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Finstall
 # Does two things:
 # 1) modifies 'ferret_paths template' to specify locations of ferret software &
diff -ru Ferret-7.6.0.orig/bin/Finstall.uses_des Ferret-7.6.0/bin/Finstall.uses_des
--- Ferret-7.6.0.orig/bin/Finstall.uses_des	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Finstall.uses_des	2024-05-23 18:16:39.228607557 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Finstall
 # Does two things:
 # 1) modifies 'ferret_paths template' to specify locations of ferret software &
diff -ru Ferret-7.6.0.orig/bin/Fpalette.csh Ferret-7.6.0/bin/Fpalette.csh
--- Ferret-7.6.0.orig/bin/Fpalette.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fpalette.csh	2024-05-23 18:16:39.239607589 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fpalette file_template    
 # determine if FERRET palette files matching template are currently on-line
 # by searching the paths in FER_DESCR
diff -ru Ferret-7.6.0.orig/bin/Fpatch.csh Ferret-7.6.0/bin/Fpatch.csh
--- Ferret-7.6.0.orig/bin/Fpatch.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fpatch.csh	2024-05-23 18:16:39.244607603 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fpatch
 # Untar's the file fer_patch.tar.Z to patch in additions to the current version
 # of ferret and its support files
diff -ru Ferret-7.6.0.orig/bin/Fpattern.csh Ferret-7.6.0/bin/Fpattern.csh
--- Ferret-7.6.0.orig/bin/Fpattern.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fpattern.csh	2024-05-23 18:16:39.252607626 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fpattern file_template    
 # determine if FERRET pattern files matching template are currently on-line
 # by searching the paths in FER_DESCR
diff -ru Ferret-7.6.0.orig/bin/Fprint_template.csh Ferret-7.6.0/bin/Fprint_template.csh
--- Ferret-7.6.0.orig/bin/Fprint_template.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fprint_template.csh	2024-05-23 18:16:39.261607652 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fprint
 # Script to translate metafiles generated by Ferret.  It runs either gksm2ps
 # (on systems where Ferret uses xgks -- releases after Mar 1994), or mtt;
diff -ru Ferret-7.6.0.orig/bin/Fpurge.csh Ferret-7.6.0/bin/Fpurge.csh
--- Ferret-7.6.0.orig/bin/Fpurge.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fpurge.csh	2024-05-23 18:16:39.270607678 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # Fpurge
 # remove all but the current version of the indicated file(s)
 
diff -ru Ferret-7.6.0.orig/bin/Fsort.csh Ferret-7.6.0/bin/Fsort.csh
--- Ferret-7.6.0.orig/bin/Fsort.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Fsort.csh	2024-05-23 18:16:39.285607722 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 # sort a list of names given as arguments into increasing
 # order based on their ~nnn~ version number extensions
 # at most one name should be without ~nnn~
diff -ru Ferret-7.6.0.orig/bin/Ftoc.csh Ferret-7.6.0/bin/Ftoc.csh
--- Ferret-7.6.0.orig/bin/Ftoc.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/Ftoc.csh	2024-05-23 18:16:39.299607762 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh -f
+#! /usr/bin/env csh -f
 #! *sh* 10/91
 #! Ftoc
 #! browse the Table of Contents of the Ferret Users Guide
diff -ru Ferret-7.6.0.orig/bin/install_ferret_links.csh Ferret-7.6.0/bin/install_ferret_links.csh
--- Ferret-7.6.0.orig/bin/install_ferret_links.csh	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/install_ferret_links.csh	2024-05-23 18:16:39.304607776 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh
+#! /usr/bin/env csh
 # install FERRET links in the /usr/local/bin area - eliminating the need to
 # modify the PATH variable
 # ver 1.0 4/92 *sh* - based on INSTALL_BIN routine
diff -ru Ferret-7.6.0.orig/bin/make_environment_tar Ferret-7.6.0/bin/make_environment_tar
--- Ferret-7.6.0.orig/bin/make_environment_tar	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/make_environment_tar	2024-05-23 18:16:39.308607788 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh
+#! /usr/bin/env csh
 
 echo ""
 if ( $#argv == 3 ) then
diff -ru Ferret-7.6.0.orig/bin/make_executable_tar Ferret-7.6.0/bin/make_executable_tar
--- Ferret-7.6.0.orig/bin/make_executable_tar	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/make_executable_tar	2024-05-23 18:16:39.313607802 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh
+#! /usr/bin/env csh
 
 # mycp needs to recursively copy subdirectories for the lib/python* directories
 set mycp = "/bin/cp -R -p"
diff -ru Ferret-7.6.0.orig/bin/make_source_tar Ferret-7.6.0/bin/make_source_tar
--- Ferret-7.6.0.orig/bin/make_source_tar	2020-06-24 19:56:01.000000000 -0230
+++ Ferret-7.6.0/bin/make_source_tar	2024-05-23 18:16:39.318607817 -0230
@@ -1,4 +1,4 @@
-#! /bin/csh 
+#! /usr/bin/env csh 
 # make_source_tar
 # ver 1.0 6/10 *acm*
 # ver 2.0 11/10 *acm* change CVS to svn
